pkg/operators/oci-handler/oci.go

```diff
@@ -503,25 +505,41 @@ func (o *OciHandlerInstance) PreStop(gadgetCtx operators.GadgetContext) error {
 }
 
 func (o *OciHandlerInstance) Stop(gadgetCtx operators.GadgetContext) error {
+	var errs []error
+
 	for _, opInst := range o.imageOperatorInstances {
 		err := opInst.Stop(o.gadgetCtx)
 		if err != nil {
-			o.gadgetCtx.Logger().Errorf("stopping operator %q: %v", opInst.Name(), err)
+			errs = append(errs, fmt.Errorf("stopping operator %q: %w", opInst.Name(), err))
```

Comment:
No need to handle this in an array, btw. Can be simplified by having `(err error)` in the signature and then just doing `err = errors.Join(err, fmt.Errorf(...))` - `errors.Join()` will return nil, if there's no error.